unittests/dataproviders/status.dataprovider.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Dataprovider;
```

Comment:
Personally I always try to have a defnitely unique Namespace. So all *my* classes start with a namespace of `Org_Heigl` to make it almost impossible to conflict with someone elses Namespace. So here *I'd* perhaps use a namespace of `ThatDamnQa\MancMetFeedTest\DataProvider` - Just to be sure that it never conflicts…

And for non-test-classes I'd then use `ThatDamnQa\MancMetFeed` …

THe added benefit would be to be able to use composer for autoloading like this:

```json
{
  "autoload" : {
    "psr-4" : {
      "ThatDamnQa\\MancMetFeed\\" : "class/"
    }
  },
  "autoload-dev" : {
    "psr-4" : {
      "ThatDamnQa\\MancMetFeedTest\\" : "unittests/"
    }
  }
}
```

That would then mean that a class `ThatDamnQa\MancMetFeed\Status` would live in file `class/Status.php` and a class `ThatDamnQa\MancMetFeedTest\DataProvider\Status` would live in file `unittests/DataProvider/Status.php`.

And you'd only need a `require_once "vendor/autoload.php";` right at the entry to your app and could eliminate all those `require`-calls ;)